kafka-client/src/main/java/dev/responsive/kafka/internal/db/CassandraKeyValueTable.java

```diff
@@ -383,6 +402,27 @@ public KeyValueIterator<Bytes, byte[]> range(
       final Bytes to,
       final long streamTimeMs
   ) {
+    return doRange(kafkaPartition, from, to, streamTimeMs, range);
+  }
+
+  @Override
+  public <PS extends Serializer<P>, P> KeyValueIterator<Bytes, byte[]> prefix(
+      final P prefix,
+      final PS prefixKeySerializer,
+      final int kafkaPartition,
+      final long streamTimeMs
+  ) {
+    final Bytes from = Bytes.wrap(prefixKeySerializer.serialize(null, prefix));
+    final Bytes to = Utils.incrementWithoutOverflow(from);
+    return doRange(kafkaPartition, from, to, streamTimeMs, this.prefix);
```

Comment:
very nitpicky but why use `range` above but `this.prefix` here?  Can only afford one `this` ? 😛

Reply:
😆 it's because this has `prefix` in the parameters so there's a scoping conflict whereas range doesn't have that. I'll use `this` for both for symmetry if you demand it haha